dashboards/Prometheus.json

```diff
@@ -5575,6 +5573,28 @@
                 "query": "pmm-server",
                 "skipUrlSync": false,
                 "type": "custom"
+            },
+            {
+                "allValue": null,
+                "current": {},
+                "datasource": "Prometheus",
+                "definition": "label_values({__name__=~\"node_time_seconds\"},job)",
+                "hide": 2,
+                "includeAll": false,
+                "label": null,
+                "multi": false,
+                "name": "scrapingInterval",
+                "options": [],
+                "query": "label_values({__name__=~\"node_time_seconds\"},job)",
+                "refresh": 2,
+                "regex": "/.*_hr-([0-9]m[0-9]s|.?[0-9]s).*/",
```

Comment:
To be clear – we can't rely on that job name format in a long run. It _will_ be broken in the future.
/cc @rnovikovP

Reply:
IMHO the only way to omit using such approach is splitting current job parameters into two parts:
job="node_exporter_agent_id_bcd0b2b6-f063-4d55-b51c-4445e6ee6165_hr"
interval="5s"

Also I don't know any other dashboard element that would require an interval parameter.

e.g.
prometheus_target_interval_length_seconds{instance="pmm-server",interval="1m0s",job="prometheus",quantile="0.99"}

node_time_seconds{...,job="node_exporter_agent_id_bcd0b2b6-f063-4d55-b51c-4445e6ee6165_hr-5s",...}

Reply:
re "It will be good to have additional metrics": let's add them to appropriate Issues in Jira to  not lost. But as I can see it's not blocking us from workaround it for now?